features/archive.feature

```diff
@@ -1116,3 +1116,62 @@ Feature: News Items Archive
         """
         {"associations": {"foo--1": {"headline": "flower", "byline": "foo", "description_text": "flower desc"}}}
         """
+
+    @auth
+    Scenario: body_html is generated from draftJS state
+        Given "archive"
+        """
+        [{"_id": "test_editor_gen_1", "guid": "test_editor_gen_1", "headline": "test"}]
+        """
+
+        When we patch given
+        """
+        {
+            "fields_meta": {
+                "body_html": {
```

Comment:
can you also add headline to tests? I wonder how it handles single line fields

Reply:
sure

Reply:
@petrjasek done